### DISCUSSION TITLE
How to Add an Avatar to My Comments?

### DISCUSSION THREAD

**Discussion by ghost** [mecha-cms/mecha#147] [Q&A]
Hi Taufik!

How can I add an Avatar to my Comments?

**Comment by taufik-nurrohman** [comment]
Avatars are generated automatically via [Gravatar](https://gravatar.com) service. This applies to comment and user files.

Making custom avatar is possible but this is not so practical. Here is an example of a user file with static avatar data:

~~~ .yaml
---
author: Alex
email: 'alex@example.com'
status: 1
avatar: '/lot/asset/png/avatar.png'
...

I am the main author of this site.
~~~

Then, you can upload a PNG image named `avatar.png` to `.\lot\asset\png` folder.

When making a comment and you are in a logged-in state, the `avatar` value will inherit to your user file.
